integration/options.go

```diff
@@ -163,6 +182,42 @@ type testOptions interface {
 
 	// VerifySeriesDebugFilePathPrefix returns the file path prefix for writing a debug file of series comparisons.
 	VerifySeriesDebugFilePathPrefix() string
+
+	// SetRepairThrottle sets minimum duration of time a namespace repair is throttled to.
+	SetRepairThrottle(throttle time.Duration) testOptions
+
+	// RepairThrottle returns the repair throttle duration.
+	RepairThrottle() time.Duration
+
+	// SetRepairInterval sets interval between repairs.
+	SetRepairInterval(interval time.Duration) testOptions
+
+	// RepairInterval returns the repair interval duration.
+	RepairInterval() time.Duration
+
+	// SetRepairTimeJitter sets the repair time jitter.
+	SetRepairTimeJitter(t time.Duration) testOptions
+
+	// RepairTimeJitter returns the repair time jitter.
+	RepairTimeJitter() time.Duration
+
+	// SetNumShards sets the number of shards.
+	SetNumShards(numShards uint32) testOptions
+
+	// NumShards returns the number of shards.
+	NumShards() uint32
+
+	// SetFetchSeriesBlocksBatchSize sets the number of Series Blocks to fetch in batch.
+	SetFetchSeriesBlocksBatchSize(n int) testOptions
+
+	// FetchSeriesBlocksBatchSize returns the number of Series Blocks to fetch in batch.
+	FetchSeriesBlocksBatchSize() int
+
+	// SetFetchSeriesBlocksBatchConcurrency sets the number of series blocks to fetch in batch concurrently.
+	SetFetchSeriesBlocksBatchConcurrency(n int) testOptions
+
+	// FetchSeriesBlocksBatchConcurrency returns the number of series blocks to fetch in batch concurrently.
+	FetchSeriesBlocksBatchConcurrency() int
```

Comment:
We currently control it by setting the [Repairer admin client](https://github.com/m3db/m3db/pull/214/files/4bf93c374fdd55cb46960d13334e205abc6eb500#diff-34fc2068cceaa93f8d6b312c44397541R270) only when the opts to enable repairs are [set](https://github.com/m3db/m3db/pull/214/files/4bf93c374fdd55cb46960d13334e205abc6eb500#diff-34fc2068cceaa93f8d6b312c44397541R160). This way the testSetup's repairer is only aware of peers when we enable repairs. We do run the repairer if those opts aren't set, but in that case, the admin client isn't aware of the peers so we don't actively perform repairs. 

We could make it short circuit the repair code path by disabling repairs in the [namespace opts](https://github.com/m3db/m3db/blob/dev/storage/namespace.go#L530-L532) too.